README.md

```diff
@@ -31,21 +31,21 @@ Get all package versions from repository:
 ```erlang
 > hex_repo:get_versions(Config).
 {ok, {200, ...,
-    #{packages => [
```

Comment:
@ericmj to recap, previously we had:

```
> hex_repo:get_versions(Config)
{ok, {200, _, #{packages => Packages}}}
```

now we have:

```
> hex_repo:get_versions(Config)
{ok, {200, _, Packages}}
```

which means we'd no longer be able to associate any "metadata" with the versions resource in a backwards compatible way (adding new keys to the return map). I think that's OK, I can't think of anything we'd use it for, but just mentioning it. Thoughts?